src/java/org/apache/cassandra/io/sstable/format/bti/BtiFormat.java

```diff
@@ -362,7 +362,7 @@ static class BtiVersion extends Version
 
             isLatestVersion = version.compareTo(current_version) == 0;
             correspondingMessagingVersion = MessagingService.VERSION_50;
-            byteComparableVersion = version.compareTo("ca") >= 0 ? ByteComparable.Version.OSS50 : ByteComparable.Version.LEGACY;
+            byteComparableVersion = version.compareTo("ca") >= 0 ? ByteComparable.Version.OSS41 : ByteComparable.Version.LEGACY;
             hasOldBfFormat = aOrLater && !bOrLater;
```

Comment:
For Review:

Most tests pass using `OSS41`, while many fail using `OSS50`, so I decided to keep the alignment with `main`.